### ISSUE TITLE
Version without ice

### ISSUE THREAD

**Issue by arporter** [arporter/psycloned_nemo#2]
I've created the `no_ice_prof` branch which is the same as the profiling branch but built with `makenemo del_key 'key_si3'` (all other build arguments remain the same).

**Comment by andmirek** [comment]
Hi Andy,
Looks I need profile_mod for compilation. Is it available on Isambard?
Mirek

**Comment by arporter** [comment]
Yes, see the README in `/projects/metoffice/nemo_gpu/PSyclone/lib/profiling`

**Comment by arporter** [comment]
Oh, and see #1 for further instructions on building with NEMO.

**Comment by andmirek** [comment]
Thanks Andy. Will you have time to connect by skype if needed during hackathon?

Best Regards,
Mirek
